ledger-service/http-json/src/main/scala/com/digitalasset/http/CommandService.scala

```diff
@@ -150,18 +151,28 @@ class CommandService(
       jwtPayload: JwtWritePayload,
       meta: Option[domain.CommandMeta],
       command: lav1.commands.Command.Command,
+      commandKind: String,
+  )(implicit
+      lc: LoggingContextOf[InstanceUUID with RequestID]
   ): lav1.command_service.SubmitAndWaitRequest = {
-
     val commandId: lar.CommandId = meta.flatMap(_.commandId).getOrElse(uniqueCommandId())
-
-    Commands.submitAndWaitRequest(
-      jwtPayload.ledgerId,
-      jwtPayload.applicationId,
-      commandId,
-      jwtPayload.actAs,
-      jwtPayload.readAs,
-      command,
+    withEnrichedLoggingContext(
+      label[lar.CommandId],
+      Map("command_id" -> commandId.toString),
     )
+      .run { implicit lc =>
+        logger.info(
+          s"Submitting $commandKind command and synchronously waiting for completion"
```

Comment:
```suggestion
          s"Submitting $commandKind command"
```
The JSON API only submits things synchronously so no point in adding that.